src/Common/src/Interop/Windows/Winsock/Interop.WSAConnect.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
@pgavlin I wouldn't revert this change since it might actually be needed by NetworkInformation but splitting to a greater granularity isn't required:

> If multiple related DllImports will all be needed by every consumer, they may be declared in the same file, named for the functionality grouping, e.g. Interop.IOErrors.cs.
(https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/interop-guidelines.md)